server/spec/entitlement_spec.rb

```diff
@@ -83,6 +83,14 @@
     @system.list_entitlements.first.quantity.should == 10
   end
 
+  it 'should allow consumption of quantity zero' do
+    pool = find_pool_for_product @virt
+    pool['consumed'].should eq(0)
+    ent = @system.consume_pool(pool.id, {:quantity => 0})
```

Comment:
Should this be an error case? I feel like zero is a special quantity, but maybe not in the "consumed" case. In any event, if this is a situation where we don't allow actually consuming less than 1, we should probably error out here rather than correcting it. Is this something that happens today?